docs/planejamento/metodologias.md

```diff
@@ -0,0 +1,84 @@
+# <a>Metodologia</a>
+
+## <a>Agile</a>
+
+<p style="text-indent: 20px; text-align: justify">
+Nosso grupo optou por utilizar as metodologias ágeis devido a sua simplicidade, e priorizando acima de tudo a comunicação entre a equipe. Dessa forma é estipulado entregas semanais, previamente decididas em reuniões, em que cada participante é delegado uma função e são decididas as tarefas de todos os membros. Seguindo os princípios do Manifesto Ágil, que defende o planejamento adaptativo por meio de Inteligência Coletiva, nosso time se organiza e focados na melhor maneira de desenvolver o projeto.
+</p>
+
+
+#### <a>Reuniões semanais</a>
+
+<p style="text-indent: 20px; text-align: justify">
+As reuniões são agendadas toda semana para dividir tarefas e discutir sobre quais são as melhores maneiras de desenvolver este projeto. Acontecerá principalmente na segunda feira e na quarta feira, de maneira assíncrona com a equipe de EPS e MDS.
+</p>
+
+<p style="text-indent: 20px; text-align: justify">
+Toda semana também, reuniremos com os clientes do Schedula em especial na quarta feira, para que o cliente esteja envolvido no ciclo de desenvolvimento do projeto. Realizando tarefas como validação, priorização e levantamento de novos requisitos.
+</p>
+
+
+
+#### <a>Criação de Issues</a>
+
+<p style="text-indent: 20px; text-align: justify">
+Após as reuniões são abertas novas Issues, listando as tarefas que devem ser concluídas, e seus respectivos prazos de entrega. Assim foram criadas novas Labels responsáveis por informar o status daquela Issue, e quando concluída é fechada. Caso o integrante responsável por quela Issue encontrar dificuldades, pode utilizar a Label "help wanted" e assim descrever a situação para que os outros participantes possam o ajudar.
+</p>
+
+<p style="text-indent: 20px; text-align: justify">
+Adotamos um template simples para a criação de Issue, no qual é necessário informar uma breve descrição da tarefa a ser realizada e uma lista contendo os objetivos. Também é passado uma lista de critérios de aceitação, para que após finalizada esta issue possa ser avaliada por outro integrante do grupo.
+</p>
+
+## <a>XP - Extreme Programming</a>
+<p style="text-indent: 20px; text-align: justify">
+Para o nosso projeto decidimos utilizar <b>Extreme Programming (XP)</b> que é uma metodologia de desenvolvimento ágil para pequenas e médias equipes que desenvolvem software baseado em requisitos vagos e que se modificam rapidamente. O <b>XP (Extreme Programming ou Programação Extrema)</b> é focada no desenvolvimento de software que possui valores e princípios, onde são fundamentados por um conjunto de práticas. Além disso, o <b>XP</b> também possui funções bem definidas para seus membros.
```

Comment:
Trocar o início do texto para:
Para o nosso projeto decidimos utilizar o Extreme Programming (XP), que é uma metodologia de desenvolvimento ágil...